actions/generate-schedule/scheduler/scheduler.go

```diff
@@ -0,0 +1,98 @@
+package scheduler
+
+import (
+	"fmt"
+	"github.com/spinnaker/rotation-scheduler/actions/generate-schedule/users"
+	"github.com/spinnaker/rotation-scheduler/proto/schedule"
+	"time"
+)
+
+const (
+	DateFormat = "Mon 02 Jan 2006"
```

Comment:
I was going to ask why you chose this date, googled it, and learned that it is the "magical reference date." 
⭐️ 📆